source/releasenotes/2025-02-13-pcc-side-by-side.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "New Side-by-Side Docs & Website Preview for Content Publisher"
+published_date: "2025-02-13"
+categories: [user-interface, content-publisher]
+---
+We introduced a new **side-by-side docs and website Preview Editor**. It is being rolled out progressively to users under a feature flag. If you use Content Publisher and don’t have access yet, contact us if interested to try it out.
```

Comment:
"Contact us" should provide a means (e.g. a link to a form or whatever) to actually contact us.

Reply:
```suggestion
We introduced a new **side-by-side docs and website Preview Editor**. It is being rolled out progressively to users under a feature flag. If you use Content Publisher and don’t have access yet, contact us in the beta users Slack channel if interested to try it out.
```